accepted/00000-auto-generate-bootstrap-repositories.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: Autogenerate Bootstrap Repositories
+- Start Date: 2020-02-19
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Automatically generate bootstrap repositories
+
+# Motivation
+[motivation]: #motivation
+
+To be able to register a system to Uyuni Server you need to have the software installed which provide the required tooling.
+
+A bootstrap repository provide the software for the supported operating systems with its dependencies. The content for the
+bootstrap repositories come from the normal channels which must be synchronized and updated on regular base.
+
+Users very often do not know this, or forget to create them when they synchronize a new product. Also the bootstrap repository
+is only updated when the user execute the generator script again.
+
+This RFC should describe how to change the current tools to automatically 
+execute the generation of bootstrap repositories after
+a new product was added and after a channel changed when a maintenance update was synced to Uyuni Server.
```

Comment:
Not just products or Maintenance Updates.

Think for example about Ubuntu, as it requires the user to add the Ubuntu main repositories, and some packages for the bootstrap repositories are taken from there.

I think it should be more: when a repository is added or when gets updates.

Reply:
This is what I mean when I write `a new product was added and after a channel changed`. On initial creation and on update.

Reply:
But it says `when a maintenance update was synced to Uyuni Server.` And a channel doesn't change only because of Maintenance Updates, as it depends on the kind of channel.

Only those from SCC change because of MUs.

I suggest the following:

`a new product was added and after a channel changed when underlying repositories synced new packages`.

Reply:
ok, will change